datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/DatasetComparisonJob.scala

```diff
@@ -47,7 +49,10 @@ object DatasetComparisonJob {
     */
   def execute(cliOptions: CliOptions, configPath: Option[String] = None)
              (implicit sparkSession: SparkSession): Unit = {
-    val config = new TypesafeConfig(configPath)
+    val config = new TypesafeConfig(configPath).validate() match {
```

Comment:
Just for future, you can write this in a much simpler way:
`val config = new TypesafeConfig(configPath).validate().get`

Reply:
This is to be explicit. Do you think it is a bad practice?